heareval/tasks/sampler.py

```diff
@@ -24,6 +24,14 @@
 import heareval.tasks.util.luigi as luigi_util
 from heareval.tasks import dcase2016_task2, nsynth_pitch, speech_commands
 
+# Currently the sampler is only allowed to run for open tasks
+# The secret tasks module will not be available for participants
+try:
+    import secret_tasks as secret_tasks
```

Comment:
oh man, the module name is secret_tasks? Why not follow the naming convention we have established in other repos:

```
hearsecrettasks
```